### PR TITLE
Updated OpenEdge Dash Documentation.

### PR DESCRIPTION
**Pull request by chris4gl**

### CODE REVIEW

**Comment by chris4gl** [comment]
I went and updated the existing OpenEdge docset as you recommended, and added specific versions for each docset. I also fixed the validation errors in the 11.6 docset, and made some more tweaks to the 11.7 docset as well.
These are available here:
https://www.progresstalk.com/docset/OpenEdge.11.5.1.docset.tgz
https://www.progresstalk.com/docset/OpenEdge.11.6.4.docset.tgz
https://www.progresstalk.com/docset/OpenEdge.11.7.2.docset.tgz
Thanks,
Chris

**Comment by Kapeli** [comment]
11.6 is still failing validation:

```
Checking OpenEdge/versions/11.6.4/OpenEdge.11.6.4.docset.tgz...
File not found at "aswsp/twophase-commit.html" for entry with row id "1080" of "OpenEdge116.docset"
```

File: docsets/OpenEdge/docset.json [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+    },
+    "aliases": ["Progress", 
+                "ABL",
+                "4GL"
+               ],
+    "specific_versions": [
+        {
+            "version": "11.7/2",
+            "archive": "versions/11.7.2/OpenEdge.11.7.2.docset.tgz"
+        }
```
Comment (Kapeli) [inline docsets/OpenEdge/docset.json]: You're missing a comma at the end of this line.

File: docsets/OpenEdge/docset.json [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
+        {
+            "version": "11.6/4",
+            "archive": "versions/11.6.4/OpenEdge.11.6.4.docset.tgz"
+        }
```
Comment (Kapeli) [inline docsets/OpenEdge/docset.json]: You're missing a comma at the end of this line.

File: docsets/OpenEdge/docset.json [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
+        {
+            "version": "11.5/1",
+            "archive": "versions/11.5.1/OpenEdge.11.5.1.docset.tgz"
+        }
```
Comment (Kapeli) [inline docsets/OpenEdge/docset.json]: You're missing a comma at the end of this line.

File: docsets/OpenEdge/docset.json (hunk 1 above)
Comment (chris4gl) [reply docsets/OpenEdge/docset.json]: Added missing commas

File: docsets/OpenEdge/docset.json (hunk 2 above)
Comment (chris4gl) [reply docsets/OpenEdge/docset.json]: Added missing commas

File: docsets/OpenEdge/docset.json (hunk 3 above)
Comment (chris4gl) [reply docsets/OpenEdge/docset.json]: Added missing commas

**Comment by chris4gl** [comment]
Updated validation error that still existed in OpenEdge 11.6.4, uploaded corrected file:

https://www.progresstalk.com/docset/OpenEdge.11.6.4.docset.tgz

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**